### PR TITLE
Audio Generation

### PR DESCRIPTION
**Pull request by AccomplishedCode**

### CODE REVIEW

**Comment by slavakurilyak** [comment]
@AccomplishedCode Thanks for this contribution! Please add usage instructions to `README.md`. Also, please add dataset instructions on how to download the 93 MIDI files.

**Comment by AccomplishedCode** [comment]
@slavakurilyak  Thanks for pointing that out! I just added a couple of updates to the README file, I hope it is up to standard!

**Comment by slavakurilyak** [comment]
@AccomplishedCode Please move your model to **/tasks/computer-vision/music-generation** folder. Create one if none exists.

File: tasks/audio/audio-generation/Music-ai/MLProject.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<<<<<<< HEAD
```
Comment (slavakurilyak) [inline tasks/audio/audio-generation/Music-ai/MLProject.yml]: @AccomplishedCode Please remove these merge comments

**Comment by slavakurilyak** [comment]
@AccomplishedCode Please delete the `mid` files since you already include download instructions to the dataset.

**Comment by slavakurilyak** [comment]
@AccomplishedCode Please revert two binary files: `ai-platform-mlflow.jpg` and `ai-platform.jpg`

**Comment by slavakurilyak** [comment]
@mlaradji Please review this pull request and assign a model UUID. As soon as it's merged, we will do a write-up (aka blog post) about this model. Thanks in advance!

**Comment by slavakurilyak** [comment]
@AccomplishedCode Please move your model to the following folder: [tasks/audio/music-generation](https://github.com/produvia/ai-platform/tree/master/tasks/audio/music-generation)

**Pull request closed without merging**